include/psp2/appmgr.h

```diff
@@ -13,57 +13,56 @@
 extern "C" {
 #endif
 
-enum {
+typedef enum SceAppMgrErrorCode {
 	//! Busy
-	SCE_APPMGR_ERROR_BUSY			= 0x80802000,
-
+	SCE_APPMGR_ERROR_BUSY               = 0x80802000,
 	//! Invalid state
-	SCE_APPMGR_ERROR_STATE			= 0x80802013,
-
+	SCE_APPMGR_ERROR_STATE              = 0x80802013,
 	//! NULL pointer
 	SCE_APPMGR_ERROR_NULL_POINTER		= 0x80802016,
-
 	//!< Invalid param
-	SCE_APPMGR_ERROR_INVALID		= 0x8080201a,
-
+	SCE_APPMGR_ERROR_INVALID            = 0x8080201a,
 	//!< Invalid self path
-	SCE_APPMGR_ERROR_INVALID_SELF_PATH	= 0x8080201e,
-
+	SCE_APPMGR_ERROR_INVALID_SELF_PATH  = 0x8080201e,
 	//!< argv is too long
-	SCE_APPMGR_ERROR_TOO_LONG_ARGV		= 0x8080201d
-};
-
-enum {
-	SCE_APPMGR_SYSTEMEVENT_ON_RESUME		= 0x10000003,
-	SCE_APPMGR_SYSTEMEVENT_ON_STORE_PURCHASE	= 0x10000004,
-	SCE_APPMGR_SYSTEMEVENT_ON_NP_MESSAGE_ARRIVED	= 0x10000005,
-	SCE_APPMGR_SYSTEMEVENT_ON_STORE_REDEMPTION	= 0x10000006,
-};
-
-enum {
-	SCE_APPMGR_INFOBAR_VISIBILITY_INVISIBLE	= 0,
-	SCE_APPMGR_INFOBAR_VISIBILITY_VISIBLE	= 1
-};
-
-enum {
-	SCE_APPMGR_INFOBAR_COLOR_BLACK	= 0,
-	SCE_APPMGR_INFOBAR_COLOR_WHITE	= 1
-};
-
-enum {
-	SCE_APPMGR_INFOBAR_TRANSPARENCY_OPAQUE	= 0,
-	SCE_APPMGR_INFOBAR_TRANSPARENCY_TRANSLUCENT	= 1
-};
+	SCE_APPMGR_ERROR_TOO_LONG_ARGV      = 0x8080201d
+} SceAppMgrErrorCode;
+
+typedef enum SceAppMgrSystemEventType {
+	SCE_APPMGR_SYSTEMEVENT_ON_RESUME             = 0x10000003,
+	SCE_APPMGR_SYSTEMEVENT_ON_STORE_PURCHASE     = 0x10000004,
+	SCE_APPMGR_SYSTEMEVENT_ON_NP_MESSAGE_ARRIVED = 0x10000005,
+	SCE_APPMGR_SYSTEMEVENT_ON_STORE_REDEMPTION   = 0x10000006
```

Comment:
ok. make sense.

if no one has more review opinions, I'll merge this. just waiting a few more hours ;)